examples/response/kb_jwt_unsecured.json

```
@@ -0,0 +1,6 @@
+{
+  "nonce": "n-0S6_WzA2Mj",
+  "aud": "https://client.example.org/cb",
+  "iat": 1709839446,
+  "sd_hash": "d-m8CkhGgx0KgSgNJvK5mR58GTR2hjhaHbBifUaPhKc"
+}
```

shouldn't this be the decoded payload of the KB-JWT from the token_response_vp_token_sd_jwt_vc.txt example just above? 
```suggestion
{
  "nonce": "n-0S6_WzA2Mj",
  "aud": "https://example.com/verifier",
  "iat": 1709838604,
  "sd_hash": "Dy-RYwZfaaoC3inJbLslgPvMp09bH-clYP_3qbRqtW4"
}
```

---

You're absolutely correct. Thanks